### DISCUSSION TITLE
Upating to latest code

### DISCUSSION THREAD

**Discussion by jkieffer0616** [Azure/ipam#149] [Q&A] [answered]
I deployed this back in March and see there have been quite a few bug fixes/enhancements added in.  Is there a good way to update to the latest code without losing the config/setup that I've done?

Just wanted to add - great job on this.  Such a simple and useful solution and I love what I've seen in it so far.

**Comment by DCMattyG** [comment, marked as answer]
Good morning @jkieffer0616, thank you so much for the kind words!

If you have done a standard deployment using our Public images (not private ACR), then you can simply restart your App Service and the new container images will be downloaded automatically.

If you are using the Private ACR option, then you can fetch/pull the latest code from this repo and push the new containers using the process from our docs page [here](https://azure.github.io/ipam/#/contributing/README?id=building-production-containers-images-using-a-private-acr)

Hope that helps, please let us know if you have any challenges and we'll get you taken care of right away!

P.S. I know that doc is in the *"contributing"* section, which isn't right. I'll get that moved to a better home right away.

**Reply by jkieffer0616** [reply]
@DCMattyG - just got a chance to follow the steps and it worked perfectly.  Thanks for pointing me to the doc!

**Reply by DCMattyG** [reply]
Happy to help, thanks so much for reaching out!
